### PR TITLE
VIP_Random Module

### PR DESCRIPTION
**Pull request by T3Marius**
Hi @partiusfabaa , i made a random vip module that gives a player a vip group from vip.json for the map or until he disconnects. the config is created in VIPCore/modules/vip_random.json

### CODE REVIEW

File: VIPCore/modules/VIP_Random/VIP_Random.cs [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+        {
+            // Handle the case where the player is not found or other exceptions
+            return false;
+        }
+    }
+
+    private CCSPlayerController? GetRandomPlayer()
+    {
+        var players = Utilities.GetPlayers()
+            .Where(p => p != null && p.IsValid && p.PlayerPawn != null && p.PlayerPawn.IsValid && !p.IsBot && !p.IsHLTV && p.Connected == PlayerConnectedState.PlayerConnected)
```
Comment (partiusfabaa) [inline VIPCore/modules/VIP_Random/VIP_Random.cs:164]: Maybe also add a check that the person is not a VIP player? 
`_vipApi?.IsClientVip(p) == false`

Comment (T3Marius) [reply VIPCore/modules/VIP_Random/VIP_Random.cs:164]: Sure

Comment (T3Marius) [reply VIPCore/modules/VIP_Random/VIP_Random.cs:147]: Done

**Comment by wohahobg** [comment]
Hello @T3Marius the module is giving this error 
16:47:28 [EROR] (cssharp:Core) Error invoking callback
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> System.NullReferenceException: Object reference not set to an instance of an object.
   at VIP_Random.VIP_Random.OnRoundStart(EventRoundStart event, GameEventInfo info) in C:\Users\Marius\Downloads\dRandomVIP-main\dRandomVIP-main\[ D3X ] Random VIP - SRC Files\VIP_Random.cs:line 75
   at InvokeStub_GameEventHandler`1.Invoke(Object, Span`1)
   at System.Reflection.MethodBaseInvoker.InvokeWithFewArgs(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   --- End of inner exception stack trace ---
   at System.Reflection.MethodBaseInvoker.InvokeWithFewArgs(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at System.Delegate.DynamicInvokeImpl(Object[] args)
   at CounterStrikeSharp.API.Core.FunctionReference.<CreateWrappedCallback>b__18_0(fxScriptContext* context) in /home/runner/work/CounterStrikeSharp/CounterStrikeSharp/managed/CounterStrikeSharp.API/Core/FunctionReference.cs:line 100
103.343750 Long frame (FreezePeriod): 20.43ms elapsed, 20.23ms sim time, 1 ticks, 6614..6614.
'Wohaho': CQ resync (catchup), client will discard cmd timing < 2185

**Pull request merged**